frontend/src/lib/components/CommandPalette/commandPaletteLogic.ts

```diff
@@ -198,6 +202,7 @@ export const commandPaletteLogic = kea<
             // eslint-disable-next-line @typescript-eslint/no-unused-vars
             const { resolver, ...cleanedCommand } = cleanedResult.source
             cleanedResult.source = cleanedCommand
+            cleanedResult.isMobile = isMobile()
```

Comment:
This is working great now on all resolutions I tried! Thank you! Again, I really appreciate you taking the time to fix this. 

The only comment I have is about this line. Where is the `isMobile` attribute on the result used?

Reply:
`cleanedResult` is only used as 'palette command executed' event properties.

Reply:
doh! should have checked one line below.